### PR TITLE
adds event for big data panel

### PR DESCRIPTION
**Pull request by fboxwala**

### CODE REVIEW

File: content/W2016/W16-event-big-data-panel.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+Title: Big Dataa Panel
+Date: 2016-02-15
+Category: Events
+Tags: big data, panels
+Slug: big-data-panel-w16
+Author: Fatema Boxwala
+Summary: Learn about the world of Big Data, and the roles of various women
```
Comment (ehashman) [inline content/W2016/W16-event-big-data-panel.md]: Summary can't be split across lines.

File: content/W2016/W16-event-big-data-panel.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+We will be taking questions from the audience at the end of the panel so
+bring any that you have.
+
+
+## Event Details ##
+
++ **Who:** Everyone!
++ **What:** Big Data Panel
++ **Where:** DC 1304
```
Comment (ehashman) [inline content/W2016/W16-event-big-data-panel.md]: That room is tiny, it only fits 50 people (and it's cramped for that number). Is that large enough for what I'm assuming is intended to be a larger event?

Comment (fboxwala) [inline content/W2016/W16-event-big-data-panel.md]: we can book a bigger room

File: content/W2016/W16-event-big-data-panel.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+Title: Big Dataa Panel
```
Comment (ehashman) [inline content/W2016/W16-event-big-data-panel.md]: As @jennyzsun mentioned, this has two a's!

**Comment by fboxwala** [comment]
Corrected room number, this should be mergeable

**Pull request merged**